package.json

```
@@ -130,6 +130,7 @@
 		"@wordpress/i18n": "^5.26.0",
 		"@wordpress/icons": "^10.10.0",
 		"@wp-playground/blueprints": "^1.1.0",
+		"@wp-playground/cli": "^1.2.3",
```

Should we update all the playground dependencies together?

* @php-wasm/node
* @php-wasm/scopes
* @php-wasm/universal
* @wp-playground/blueprints
* @wp-playground/wordpress

```suggestion
		"@wp-playground/blueprints": "^1.2.3",
		"@wp-playground/cli": "^1.2.3",
```

---

Done!